server/bundler/bin/bundler.js

```diff
@@ -21,9 +21,12 @@ var webpackConfig = require( process.cwd() + '/webpack.config' ),
  * Variables
  */
 var start = new Date().getTime(),
-	bundleEnv = config( 'env' ),
 	outputOptions;
 
+if ( 'development' === config( 'env' ) ) {
```

Comment:
It looks like `bundleEnv` wasn't used at all before. Why do we want to exit early for `development` end?